### PR TITLE
Migrate from bazel-distribution to rules_jvm_external for Maven distribution

### PR DESCRIPTION
**Pull request by sugarmanz**

### CODE REVIEW

File: maven/workspace.bzl [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
         http_archive,
         name = "rules_jvm_external",
         strip_prefix = "rules_jvm_external-%s" % tag,
         sha256 = sha,
         url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % tag,
     )
+
+# Load maven rules from git
+def maven_repository(
+        remote = "https://github.com/sugarmanz/rules_jvm_external.git",
```
Comment (brocollie08) [inline maven/workspace.bzl:16]: plan is to try to get your PR in `rules_jvm_external` merged, yea?

Comment (sugarmanz) [reply maven/workspace.bzl:16]: Ideally, but IMO it's not really acceptable as-is. It requires an `--experimental_google_legacy_api` flag, which would require every consumer of the rules to enable that flag. I'm not sure what the path forward is, but will likely try to get some feedback from the team over there.

Comment (sugarmanz) [reply maven/workspace.bzl:16]: Opened a PR for feedback:
https://github.com/bazelbuild/rules_jvm_external/pull/1049

**Review by brocollie08 — APPROVED** [review]

**Comment by intuit-svc** [comment]
:rocket: PR was released in [`v0.12.0`](https://github.com/player-ui/rules_player/releases/tag/v0.12.0) :rocket:

**Pull request merged**